lib/dry/schema/message_set.rb

```diff
@@ -129,14 +38,14 @@ def messages_map(messages = self.messages)
 
       # @api private
       def paths
-        @paths ||= messages.map(&:path).uniq
+        @paths ||= errors.map(&:path).uniq
       end
 
       # @api private
       def initialize_placeholders!
         return @placeholders = EMPTY_HASH if empty?
 
-        @placeholders = paths.reduce(EMPTY_HASH.dup) do |hash, path|
+        @placeholders ||= paths.reduce(EMPTY_HASH.dup) do |hash, path|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Memoized variable `@placeholders` does not match method name `initialize_placeholders!`. Use `@initialize_placeholders` instead.](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764391)